src/Sylius/Bundle/AttributeBundle/Form/Type/AttributeType/Configuration/OptionListAttributeConfigurationType.php

```diff
@@ -21,8 +21,8 @@
  */
 class OptionListAttributeConfigurationType extends AbstractType
 {
-    const FORMAT_DROPDOWN           = 'dropdown';
-    const FORMAT_MULTIPLE_SELECTION = 'multiple_selection';
+    const DROPDOWN    = 'dropdown';
```

Comment:
```
const SINGLE_VALUE = 'single_value'
const MULTI_VALUE = 'multi_value'
```

We might have to run a DB migration to update these settings on existing attributes :/  But I think we should do it for clarity.